src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/DotNetXamarinProject.cs

```diff
@@ -29,7 +29,6 @@ protected DotNetXamarinProject (string debugConfigurationName = "Debug", string
 				SetProperty ("TargetFramework", "net6.0-android");
 				SetProperty ("EnableDefaultItems", "false");
 				SetProperty ("AppendTargetFrameworkToOutputPath", "false");
-				SetProperty ("AppendRuntimeIdentifierToOutputPath", "false");
```

Comment:
I suspect this will break random tests, and I'll need to update them. We'll see.